.github/workflows/js_test.yml

```diff
@@ -110,3 +110,5 @@ jobs:
         run: yarn run check-version
       - name: Test
         run: yarn run test
+      - name: Test: Vitest
+        run: yarn run test:vitest
```

Comment:
AI SDK's mock providers don't seem to support Jest anymore

We should honestly get everything over to Vitest at some point anyway, but doing this as a stopgap